recipes/dfp/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import copy, get
+
+required_conan_version = ">=1.54.0"
+
+
+class DfpConan(ConanFile):
+    name = "dfp"
+    package_type = "library"
+
+    # Optional metadata
+    description = "Decimal Floating Point Arithmetic Library"
+    homepage = "https://github.com/epam/DFP/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ("Apache-2.0", "Intel")
+    topics = ("decimal", "dfp", "ieee-754", "deltix")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+
+    # Sources are located in the same place as this recipe, copy them to the recipe
+    exports_sources = "CMakeLists.txt", "src/*", "include/*"
+
+    def source(self):
```

Comment:
In the upstream CMakeLists.txt, [you've set C++ 11 as the minimum required version ](https://github.com/epam/DFP/blob/master/native/CMakeLists.txt#L77) - You'll want to add [something like this check in the `validate()` method](https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L82) to ensure C++11 is used

Reply:
You right - this is just part of one for old version.
This library do not requires C++ at all.
I have pushed fix.